tests/v2/test_waku_rln_relay_onchain.nim

```diff
@@ -104,6 +106,31 @@ proc createEthAccount(): Future[(keys.PrivateKey, Address)] {.async.} =
   return (pk, acc)
 
 procSuite "Waku-rln-relay":
+
+  ########################
+  ## Ganache installation
+  ########################
```

Comment:
Can you wrap the ganache install and start logic in a `proc` (e.g., `proc setupGanache()`) and call it here?

Reply:
+1